nncf/common/quantization/initialization/range.py

```diff
@@ -204,3 +207,32 @@ def use_means_of_mins(self) -> bool:
     @property
     def use_means_of_maxs(self) -> bool:
         return not self._is_weights and not self._is_per_channel
+
+    def get_reduction_aggregation_axes(
+        self, shape_to_reduce: Union[Tuple[int], List[int]], quantization_axes: Union[Tuple[int], List[int]]
```

Comment:
Can we use `Tuple[int]` everywhere?

Reply:
the function works as expected for list and tuples. What do you mean?

Reply:
I mean can we streak all backends to use only tuples?

Reply:
If there is a reason for it - I think yes. If you are sure that it should be done, please create a ticket. This PR is not intended to implement this restriction

Reply:
Please estimate efforts to restrict all backends to tuples. I feel it just a couple lines of code